### DISCUSSION TITLE
12/13 유승범 블로그 1회차

### DISCUSSION THREAD

**Discussion by YuSeungBeom** [Orm-camp/blog-feedback#53] [동글동글 4기 피드백]
https://danny4558.tistory.com/1

**Comment by Jungminchae** [comment]
- __init__(self) 보다는 __init__ 생성자라고 표현하시면 어떨까요? init을 보통 생성자라고 많이 부릅니다.
- 제목에 (self)에서 self를 빼시고 self와 관련한 소제목을 만들어 그것에 대해 다뤄주시면 더 좋을 것 같습니다.
- self의 중요성: -> 여기에서 :를 빼시는 것은 어떨까요 : 은 보통 :을 쓰고 그 다음에 설명을 쭉 적을때 사용하다보니 뭔가 어색해 보입니다.
- 마지막에 결론을 적어서 한 번 정리해 주신 것 아주 좋다고 생각합니다.
- 설명을 아주 잘하시네요. 첫 공유 감사합니다.

**Reply by YuSeungBeom** [reply]
피드백 감사합니다

**Comment by smahn9123** [comment]
티스토리에 코드 블럭을 직접 복사붙여넣을 수 있는 옵션이 있는데, 이걸 쓰면 코드 이미지 캡쳐를 따로 안하셔도 될 것 같습니다!
개념 설명 잘해주셔서 많이 도움되었습니다.

**Comment by devSeyong** [comment]
class 강의 들을 때 잘 이해가 되지 않았던 부분인데 이렇게 정리해서 올려주셔서 감사합니다! 잘 읽고 공부해보도록 하겠습니다!
